Wheres-Waldo-Frontend/src/components/HomePage.jsx

```diff
@@ -0,0 +1,132 @@
+import { Link } from "react-router-dom";
+import waldoBeach from "../assets/waldoBeach.jpg";
+import waldoDowntown from "../assets/waldoDowntown.jpg";
+import waldoFactory from "../assets/waldoFactory.jpg";
+import styled from "styled-components";
+import styles from "./HomePage.module.css";
+
+function HomePage() {
+	return (
+		<>
+			<Header />
+			<Body />
+		</>
+	);
+}
+
+function Header() {
+	const title = "Where's Waldo! A photo tagging game";
+	return (
+		<>
+			<div className="title">
+				<h1 role="title">{title}</h1>
```

Comment:
Don't think you need the `title` const, I'd just put the text in here directly, ie 
```
<h1 role="title">Where's Waldo! A photo tagging game</h1>
```
Don't need the fragment either